### ISSUE TITLE
Question about the training data

### ISSUE THREAD

**Issue by YujieOuO** [alibaba/animate-anything#22]
Thanks for providing animate_anything_svd_v1.0 model. 

May i ask how many videos are used to finetune the svd model to acquire animate_anything_svd_v1.0 ?
and maybe the different between svd and svd_mask model is the conv_in module? for the input dimension from 8 to 9.

**Comment by daizuozhuo** [comment]
I use 100k clips for finetuning.

**Comment by YujieOuO** [comment]
> I use 100k clips for finetuning.


Sorry to bother you again, and I'd like to know if the 100k data used to train SVD-mask is from webvid10M?
